index.js

```diff
@@ -362,6 +364,14 @@
         }
       });
 
+  //  keys :: StrMap a -> Array String
+  var keys = function(strMap) {
+    if (typeof Object.keys === 'function') return Object.keys(strMap);
```

Comment:
Is this for performance? My default position is to minimize branching, but I'm happy to introduce branching if there's a significant performance difference.